benchmarks/bench_context_manager_overhead.py

```diff
@@ -24,5 +24,5 @@
         pass
     timings.append(time.time() - t)
 
-print(f"Overhead per call: {mean(timings) * 1e3:.3f} "
-      f"+/-{stdev(timings) * 1e3:.3f} ms")
+print("Overhead per call: {:.3f} +/-{:.3f} ms"
+      .format(mean(timings) * 1e3, stdev(timings) * 1e3))
```

Comment:
fstrings are python 3.6+
since we support 3.5+ we might has well support it in the benchmarks